packages/react-components/components/button/tests/chromatic/button.chroma.jsx

```diff
@@ -74,7 +75,7 @@ stories()
              <div className="flex flex-row">
                  <div className="flex flex-column items-start">
                      <Button.Group>
-                         <Button>One</Button>
+                         <Button>One <Label size="mini">12</Label></Button>
```

Comment:
I though that a button would enforce the size of the label based on is own size. I'm I too early, it's not done yet?

Reply:
How would you do this? Pass a prop down the tree?

Reply:
This is easy to do with `cloneElement`

Reply:
Extracted in a separate issue: https://github.com/gsoft-inc/sg-orbit/issues/156